src/v/utils/tests/token_bucket_test.cc

```diff
@@ -178,3 +178,16 @@ SEASTAR_THREAD_TEST_CASE(test_available_tokens_capacity) {
     consume(throttler, 10);
     BOOST_REQUIRE_EQUAL(throttler.available(), RATE * 3 - 10);
 }
+
+SEASTAR_THREAD_TEST_CASE(test_available_tokens_change_capacity) {
```

Comment:
nit: could we squash this commit with the one that introduced the `update_capacity` method?

Reply:
I want to have tests and code in different commits